addons/l10n_dk_nemhandel/views/account_journal_dashboard_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_journal_dashboard_kanban_view" model="ir.ui.view">
+        <field name="name">account.journal.dashboard.kanban</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.account_journal_dashboard_kanban_view" />
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//kanban" position="inside">
+                    <field name="is_nemhandel_journal" invisible="1"/>
+                    <field name="l10n_dk_nemhandel_proxy_state" invisible="1"/>
+                </xpath>
+
+                <xpath expr="//t[@id='account.JournalBodySalePurchase']//div" position="inside">
+                    <t t-if="record.l10n_dk_nemhandel_proxy_state.raw_value === 'receiver'">
```

Comment:
i suppose that if the proxy_state is set then it means that the company is from DK ? or every company can send though nemhandel ?

Reply:
I will hide the res_config_setting if they're not danish. (and they needed a CVR anyway to be registered)